config/vufind/config.ini

```diff
@@ -2098,6 +2098,10 @@ related[] = "Similar"
 ; records). Default is false.
 ;alwaysDisplayIndexRecordInStaffView = false
 
+; If set to "marc", the subject headings will be displayed in the order they are saved,
+; otherwise they are sorted numerically
+;subjectHeadingsSort = "marc"
```

Comment:
It occurs to me that it might make sense to rename this heading to be more MARC-specific, since different formats might require different settings, and this setting truly applies only to MARC records. What if you changed it to something like this:

```
; This setting controls the display order of subject headings from MARC records.
; If set to "record" it will retain the order of headings from the MARC record; if
; set to "field" (the default), it will sort numerically by MARC tag.
;marcSubjectHeadingsSort = "record"
```

I think there might also be an argument for making the new behavior the default, since it probably makes more sense to represent the record as-written rather than sorting the subject headings by type. I could seek more opinions on that if you think it's worth considering.

Reply:
I like the suggested config changes.

I think it would make sense to have `record` be the default too. In our case the catalogers add the subjects in a specific order and want them displayed in that order; so I imagine that is the case with others as well. I'm not opposed to leaving `field` as the default though if we're worried others wouldn't want the order to change from they way they have it without a conscious config change.

Reply:
Let's go ahead and change the default. I'll put a note in the changelog indicating how to get back to the old behavior; it's an easy fix if anyone cares, but I think for most, this will either make no difference or be an improvement.

Reply:
I realised when pushing you made the modifications and I merged them including a fix for an unexpected behaviour with `$this->mainConfig->Record->marcSubjectHeadingsSort ?? '' === 'field'`